Tools/scripts/serve.py

```diff
@@ -25,7 +25,7 @@ def app(environ, respond):
         return [b'not found']
 
 if __name__ == '__main__':
-    path = sys.argv[1]
+    path = sys.argv[1] if len(sys.argv) > 1 else os.getcwd()
     port = int(sys.argv[2]) if len(sys.argv) > 2 else 8000
     httpd = simple_server.make_server('', port, app)
     print("Serving {} on port {}, control-C to stop".format(path, port))
```

Comment:
When I press CTRL+C, I get a ResourceWarning. Please add httpd.server_close() after print("\b\bShutting down."). By the way, I'm not sure that "\b" is supported on Windows. I suggest to remove ""\b\b" to make the script more portable.

Reply:
Fixed